bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/dtree/inst/Alopeke.scala

```diff
@@ -0,0 +1,20 @@
+package edu.gemini.model.p1.dtree.inst
+
+import edu.gemini.model.p1.dtree._
+import edu.gemini.model.p1.immutable._
+
+object Alopeke {
+
+  def apply() = new ModeNode
+
+  class ModeNode extends SingleSelectNode[Unit, AlopekeMode, AlopekeBlueprint](()) {
+    override val title = "Mode"
+    override val description = "Please select mode, pixel scale, and field of view."
```

Comment:
Do you pick all the choices at once (in one step)?

Reply:
Yes. All the choices are determined by the mode that you pick.